benchmarks/common/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 # This file is part of covfie, a part of the ACTS project
 #
-# Copyright (c) 2022 CERN
+# Copyright (c) 2022-2024 CERN
 #
 # This Source Code Form is subject to the terms of the Mozilla Public License,
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at http://mozilla.org/MPL/2.0/.
 
-add_library(benchmark test_field.cpp)
+add_library(covfie_benchmark test_field.cpp)
 
-target_include_directories(benchmark PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(covfie_benchmark PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-target_link_libraries(benchmark PUBLIC core)
+target_link_libraries(covfie_benchmark PUBLIC covfie::core)
 
-target_compile_definitions(benchmark PRIVATE _CRT_SECURE_NO_WARNINGS)
+target_compile_definitions(covfie_benchmark PRIVATE _CRT_SECURE_NO_WARNINGS)
```

Comment:
I'm not on board with the benchmarks, tests, and examples being renamed; these should never be installed anyway, so there is no way for any name collisions to occur.